compiler/docgen.nim

```diff
@@ -873,7 +873,7 @@ proc generateTags*(d: PDoc, n: PNode, r: var Rope) =
 
 proc genSection(d: PDoc, kind: TSymKind) =
   const sectionNames: array[skTemp..skTemplate, string] = [
-    "Exports", "Imports", "Types", "Vars", "Lets", "Consts", "Vars", "Procs", "Funcs",
+    "Exports", "Dependencies", "Types", "Vars", "Lets", "Consts", "Vars", "Procs", "Funcs",
```

Comment:
Hum, I liked "imports" better...

Reply:
Eh, you were the one who proposed "Dependencies" :)

RFC? :rofl:

Reply:
I guess this means I changed my mind...